docs/src/development/ssh/troubleshoot-ssh.md

```diff
@@ -30,6 +30,11 @@ git push origin main
 
 Make sure your public key has been uploaded to your user account. Check it in the [Platform.sh console](https://console.platform.sh/).
 
+A given SSH key pair can only be linked to a single user account.
+
+If you add an already used SSH key to another account, you see the error: `SSH key can not be duplicated`.
+Generate a new pair of SSH keys for the second user account you want to add.
```

Comment:
This is a different error than the above and the solution is different.. I'd suggest a separate heading for it.